src/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -243,12 +238,133 @@ def create(style_dataset, content_dataset, style_feature=None,
     cuda_gpus = _mxnet_utils.get_gpus_in_use(max_devices=params['batch_size'])
     num_mxnet_gpus = len(cuda_gpus)
 
+    style_sa = style_dataset[style_feature]
+    idx_column = _tc.SArray(range(0, style_sa.shape[0]))
+    style_sframe = _tc.SFrame({"style": idx_column, style_feature: style_sa})
+
     if verbose:
         # Estimate memory usage (based on experiments)
         cuda_mem_req = 260 + batch_size_each * 880 + num_styles * 1.4
 
         _tkutl._print_neural_compute_device(cuda_gpus=cuda_gpus, use_mps=False,
                                             cuda_mem_req=cuda_mem_req, has_mps_impl=False)
+
+    content_images_loader = _SFrameSTIter(content_dataset, batch_size, shuffle=True,
+                                  feature_column=content_feature, input_shape=input_shape,
+                                  loader_type=content_loader_type, aug_params=params,
+                                  sequential=params['sequential_image_processing'])
+
+    use_tf = params['use_tensorflow']
+    rs = _np.random.RandomState(1234)
```

Comment:
Isn't this specific to the TF case?

Reply:
Not entirely. It's shared between MxNet and TF.

We call random indices in MxNet as well 

Line 423:

`indices = _mx.nd.array(rs.randint(num_styles, size=batch_size_each), dtype=_np.int64, ctx=c.context)`